src/constant_data/aliases.json

```diff
@@ -0,0 +1,326 @@
+[

```

Comment:
This can get messy real fast (especially if stuff gets out of order alphabetically. Maybe format as:

```
{
    "id": 1,
    "aliases": [ ... ]
}
```